articles/architecture-scenarios/b2c/branding.md

```diff
@@ -0,0 +1,121 @@
+
+### Change Password
+
+The Change Password page is used whenever a user takes advantage of the [Password Reset](#) functionality. Like the Login Page, you can customize the Change Password page to reflect your organization's branding. 
+
+::: panel Content Delivery Networks
+If you're using custom Login and/or Change Password pages, we recommend using an external Content Delivery Network (CDN) to help speed up page load and browser rendering time for your pages.
+:::
+
+### Errors
+
+If there are issues with user sign up or login, Auth0 provides error messages that indicate what the problem is under the hood. However, the default messages are somewhat cryptic, especially to the end user, since it will likely be missing context that only you can supply. As such, we recommend customizing your error pages to provide the missing context-specific information directly to your users.
+
+Furthermore, customizing your error pages allows you to display your branding, not Auth0's, as well as provide useful information to your users as to what should be done next. This information might include a link to a FAQ or how to get in touch with your company's support team or help desk.
+
+At the minimum, we recommend that you provide your visitors with:
+
+* Visual indicators like a friendly-name and your logo for display
+* An email address and URL for your organization's Support team
+
+Auth0 does not provide an out-of-the-box user interface for customizing your Error Pages, but you can use the [Tenant Settings endpoint of the Management API](/api/management/v2#!/Tenants/patch_settings) to configure them. Alternatively, if you can create and host your Error Page then you can have Auth0 direct users to that page instead of using the Auth0-hosted option. 
+
+
+### Custom Domains
+
+By default, the URL associated with your tenant will include its name and possibly a region-specific identifier. For example, tenants based in the US have the URL **example.auth0.com** while those based in Europe have **example.eu.auth0.com**.
+
+If you want to use a name that is consistent with your company's brand, you can configure a [custom domain](/custom-domains). Each tenant can support one custom domain name.
+
+Moreover, the custom domain functionality offers you complete control over the [certificate management](/custom-domains#certificate-management) process. By default, Auth0 provides standard SSL certificates, but if you configure a custom domain, you can use Extra Validation (EV) SSL certificates or similar to provide the visual, browser-based cues that offer your visitors additional peace of mind.
+
+In general, we see customers having the most success when they use a centralized domain for authentication -- this is especially the case if the company offers multiple products or service brands. By using a centralized domain, you can provide end users with a consistent user experience while also minimizing the need to maintain multiple production tenants in Auth0.
```

Comment:
```suggestion
In general, we see customers having the most success when they use a centralized domain for authentication -- this is especially the case if the company offers multiple products or service brands. By using a centralized domain, you can provide end users with a consistent user experience whilst also minimizing the need to maintain multiple production tenants in Auth0.
```

Reply:
"While" is a more modern construction. "Whilst" is somewhat dated.